Assets/Scripts/StaminaBar.cs

```diff
@@ -31,6 +34,7 @@ public bool move()
         if(stamina > 0)
         {
             stamina--;
+            staminaBarDisplay.ChangeSprite(stamina, maxStamina);

```

Comment:
this is bad oop, the stamina bar should be able to work even without a staminaBarDisplay. This makes them coupled to each other